Documentation/api.md

```diff
@@ -320,6 +320,7 @@ StorageSpec defines the configured storage for a group Prometheus servers.
 | Field | Description | Scheme | Required |
 | ----- | ----------- | ------ | -------- |
 | class | Name of the StorageClass to use when requesting storage provisioning. More info: https://kubernetes.io/docs/user-guide/persistent-volumes/#storageclasses DEPRECATED | string | true |
+| emptyDir | EmptyDirVolumeSource to be used by the Prometheus StatefulSets. If specified, used in place of any volumeClaimTemplate. More info: https://kubernetes.io/docs/concepts/storage/volumes/#emptydir | *v1.EmptyDirVolumeSource | false |
```

Comment:
Just noticed this. How do I get the doc generator to link this `*v1.EmptyDirVolumeSource ` to `https://v1-6.docs.kubernetes.io/docs/api-reference/v1.6/#emptydirvolumesource-v1-core` ?

Reply:
you need to add the link to the mapping at the beginning of the file

https://github.com/coreos/prometheus-operator/blob/6bfeae2e69e8ce438963345d63a290082ecdd9b5/cmd/po-docgen/api.go#L42-L50

it's a bit tribal, but it's gotten us far :slightly_smiling_face: , maybe one day I'll find some time to automate that table away